src/scripts/GameController.ts

```diff
@@ -127,6 +127,8 @@ class GameController {
     static addKeyListeners() {
         // Oak Items
         const $oakItemsModal = $('#oakItemsModal');
+        $oakItemsModal.data('allowToggle', true);
```

Comment:
Just for consistency with the other areas, can we change this to be:
```suggestion
        $oakItemsModal.data('allow-toggle', true);
```

Reply:
The way this works is by setting that value to `false` when toggling the modal, and then waiting for the event `hidden.bs.modal` or `shown.bs.modal` until we allow toggling again. I'm pretty sure we need to set the initial value to `true`, otherwise we need to open the modal manually at least once before we can use the hotkey.

Reply:
It might be better to instead flip the flag, so its `disable-toggle`?
That way you don't need anything there initially?

Reply:
Done.